desktop-widgets/mainwindow.cpp

```diff
@@ -292,6 +292,9 @@ MainWindow::MainWindow() : QMainWindow(),
 	connect(ui.profTankbar,        &QAction::triggered, sWrapper->techDetails, &TechnicalDetailsSettings::setTankBar);
 	connect(ui.profTissues,        &QAction::triggered, sWrapper->techDetails, &TechnicalDetailsSettings::setPercentageGraph);
 
+	connect(ui.profTissues,        &QAction::triggered, [=]() { ui.profHR->setChecked(false); sWrapper->techDetails->setHRgraph(false); });
+	connect(ui.profHR,             &QAction::triggered, [=]() { ui.profTissues->setChecked(false); sWrapper->techDetails->setPercentageGraph(false); });
+
```

Comment:
the policy nowadays seems to be C++ lambdas are fine as long as they are simple.
so this is OK.

Reply:
Careful about lifetime management! With lambdas, Qt will not be able to remove the connection if one of the objects is destroyed.

Here, I'm not even sure how this works. It seems like the whole `ui` object copied for each lambda. Sounds like a very bad idea.

Better turn this into two times two `connect()`s or into a member function.

Reply:
policy is that I continue to absolutely HATE AND DESPISE lambdas, but that some very talented developers got suckered into thinking they are useful and that I don't see the point in arguing with them (and potentially offend good contributors).
So whenever we can not use them, please let's not use them. ESPECIALLY if the developer using them says "I don't understand how this works". That's a clear sign that we SHOULD DO THIS DIFFERENTLY.